tickets-test/src/main/java/no/unit/nva/publication/ticket/test/TicketTestUtils.java

```diff
@@ -112,6 +120,23 @@ public static Publication createPersistedPublication(PublicationStatus status, R
         return persistPublication(resourceService, publication);
     }
 
+    public static Publication createPersistedPublishedPublicationWithUnpublishedFilesAndContributor(URI userCristinId,
+                                                                                                    ResourceService resourceService)
+        throws ApiGatewayException {
+        var publication = randomPublication().copy()
+                              .withEntityDescription(randomEntityDescription(JournalArticle.class))
+                              .withStatus(PUBLISHED)
+                              .withAssociatedArtifacts(new AssociatedArtifactList())
+                              .build();
+
+        var identity = new Identity.Builder().withName(randomString()).withId(userCristinId).build();
+        var contributor = new Contributor.Builder().withIdentity(identity).withRole(new RoleType(Role.CREATOR)).build();
+        var entityDesc = publication.getEntityDescription().copy().withContributors(List.of(contributor)).build();
+        var publicationWithContributor = publication.copy().withEntityDescription(entityDesc).build();
+
+        return persistPublication(resourceService, publicationWithContributor);
+    }
+
```

Comment:
Not sure why this method is in TICKET testUtils. Is there somewhere else we can put it?